samcli/lib/hook/hook_wrapper.py

```diff
@@ -80,11 +80,9 @@ def prepare(
             "IACProjectPath": iac_project_path if iac_project_path else str(Path.cwd()),
             "OutputDirPath": output_dir_path,
             "Debug": debug,
+            "Profile": aws_profile,
+            "Region": aws_region,
```

Comment:
This change isn't necessary. It was needed when I had some dataclasses that I ended up removing. However, I wonder if there's much difference ( e.g. `params.get("Profile")` would be `None` in either case anyway)